### PR TITLE
Remove reorgs route and link from web interface.

### LINKED ISSUE #322: Remove /reorgs URL
The URL https://explorer.celo.org/reorgs should not exist, as reorgs are not possible on Celo.

### PR DESCRIPTION
**Pull request by rkachowski**
Removes block reorg link and web route from the application to prevent user confusion (Celo doesn't have reorgs).

Closes #322

### CODE REVIEW

**Comment by martinvol** [comment]
Does it make sense to delete the code of the view too?

**Comment by rkachowski** [comment]
There's no reorg view, it's just the standard block view with reorg flavoring. We can remove the reorg code from the controller though.

File: apps/block_scout_web/lib/block_scout_web/web_router.ex [hunk 1]
```diff
@@ -39,8 +39,6 @@ defmodule BlockScoutWeb.WebRouter do
       resources("/signers", BlockSignersController, only: [:index], as: :signers)
     end
 
-    get("/reorgs", BlockController, :reorg, as: :reorg)
-
     get("/uncles", BlockController, :uncle, as: :uncle)
```
Comment (carterqw2) [inline apps/block_scout_web/lib/block_scout_web/web_router.ex:42]: Perhaps, we should remove uncles as well?

**Review by carterqw2 — APPROVED** [review]
Looks good. There is an issue with gettext though.

**Pull request merged**